### PR TITLE
Template instantiate SetFunctionBasedPath to remove spurious classes

### PR DESCRIPTION
**Pull request by aymanhab**
Fixes issue #0

### Brief summary of changes
template instantiate SetFunctionBasedPath in interface file so swig results are usable in scripting and not bloated with unusable classes
### Testing I've completed
None other than the change/fix f=to the java produced classes

### CHANGELOG.md (choose one)

- no need to update because internal

- - -
This change is [Reviewable](https://reviewable.io/reviews/opensim-org/opensim-core/3608)

### CODE REVIEW

**Comment by aymanhab** [comment]
@nickbianco this was necessary to make the ModelFactory method usable

File: Bindings/actuators.i [hunk 1]
```diff
@@ -12,6 +12,7 @@
 %include <OpenSim/Actuators/Millard2012AccelerationMuscle.h>
 %include <OpenSim/Actuators/McKibbenActuator.h>
 %include <OpenSim/Actuators/DeGrooteFregly2016Muscle.h>
+%template (SetFunctionBasedPaths) OpenSim::Set<OpenSim::FunctionBasedPath>;
```
Comment (nickbianco) [inline Bindings/actuators.i:15]: To follow the naming convention of other sets --> `FuctionBasedPathSet`?

**Review by nickbianco — CHANGES_REQUESTED** [review]
One minor suggestion for the set name.

File: Bindings/actuators.i (hunk 1 above)
Comment (aymanhab) [reply Bindings/actuators.i:15]: In the past, naming for classes made up for swig interface started with Set while ones that existed in C++ used the C++ name which ended in Set, so we have the mix but if you feel strongly about it or plan to typedef in C++ I will make the change so let me know.

Comment (nickbianco) [reply Bindings/actuators.i:15]: Ah okay, I was mixing up name conventions. One more question, should the template name be pluralized, or just be `SetFunctionBasedPath`?

Comment (aymanhab) [reply Bindings/actuators.i:15]: I went back and forth on that and you'll see I initially had it without but looking at simulation.i that we most often use plurals
https://github.com/opensim-org/opensim-core/blob/7703592938979b54dc78ef4561781bbb7cc4994a/Bindings/simulation.i#L149

**Review by nickbianco — APPROVED** [review]

**Comment by aymanhab** [comment]
Thanks @nickbianco 🥇

**Pull request merged**